nodejs/webrtc-hello-world/src/server.ts

```diff
@@ -17,9 +17,9 @@ const voiceApplicationPhoneNumber = <string>process.env.VOICE_APPLICATION_PHONE_
 const voiceApplicationId = <string>process.env.VOICE_APPLICATION_ID;
 const voiceCallbackUrl = <string>process.env.VOICE_CALLBACK_URL;
 const outboundPhoneNumber = <string>process.env.OUTBOUND_PHONE_NUMBER;
-
 const callControlUrl = `${process.env.BANDWIDTH_WEBRTC_CALL_CONTROL_URL}/accounts/${accountId}`;
-const sipxNumber = <string>process.env.BANDWIDTH_WEBRTC_SIPX_PHONE_NUMBER;
+
+var webRTCController = bandwidthWebRTC.APIController;
```

Comment:
Any reason this is a `var` vs a `const` or at least a `let`? Also purely preference, but I think I'd prefer `Rtc` capitalization to `RTC`.

Reply:
I lifted the code from another app, so no reason. I'll update.